inference-engine/ie_bridges/java/org/intel/openvino/Data.java

```diff
@@ -0,0 +1,8 @@
+package org.intel.openvino;
+
+public class Data extends IEWrapper{
+
+    protected Data(long addr) {
+        super(addr);
+    }
```

Comment:
looks quite simple, where is it used?

Reply:
We creates ```Data``` objects in```getOutputsInfo()```  method of ```CNNNetwork``` class. 
This constructor is called from C ++ code.

Reply:
Is that answered?

Reply:
yes